tests/System.Text.Primitives.Tests/Formatting/TimeTests.cs

```diff
@@ -51,6 +53,7 @@ public void RandomDateTimeTests(char format)
         static void TestDateTimeFormat(DateTime dt, char format, SymbolTable symbolTable)
         {
             var expected = dt.ToString(format.ToString(), CultureInfo.InvariantCulture);
+            if (format == 'r') expected = expected.ToLowerInvariant();
```

Comment:
Is that the difference between 'r' and 'R'?
I thought both 'r' and 'R' produce the same result.

```C#
DateTime date1 = new DateTime(2010, 8, 18);
Console.WriteLine(date1.ToString("r", CultureInfo.InvariantCulture)); \\ Output: Wed, 18 Aug 2010 00:00:00 GMT
Console.WriteLine(date1.ToString("R", CultureInfo.InvariantCulture)); \\ Output: Wed, 18 Aug 2010 00:00:00 GMT
```